src/Recordings.cpp

```diff
@@ -323,6 +323,7 @@ bool Recordings::UpdatePvrRecording(const tinyxml2::XMLNode* pRecordingNode, kod
 
   tag.SetSeriesNumber(PVR_RECORDING_INVALID_SERIES_EPISODE);
   tag.SetEpisodeNumber(PVR_RECORDING_INVALID_SERIES_EPISODE);
+  tag.SetEpisodePartNumber(PVR_RECORDING_INVALID_SERIES_EPISODE);
```

Comment:
That's actually not needed. Default ctor of PVRReocrding class will do that. Same for line 324 and 325. Not a bug, though. Just efficiency of code.

Reply:
Removed all three